sdk/keyvault/Azure.Security.KeyVault.Administration/api/Azure.Security.KeyVault.Administration.netstandard2.0.cs

```diff
@@ -61,6 +136,13 @@ public partial class KeyVaultPermission
         public System.Collections.Generic.IReadOnlyList<string> NotActions { get { throw null; } }
         public System.Collections.Generic.IReadOnlyList<string> NotDataActions { get { throw null; } }
     }
+    public partial class KeyVaultServiceError
```

Comment:
We usually don't expose these.

Reply:
The reason I did is because it is part of the `FullBackupDetails` and `FullResteoreDetails`. What is the correct way to expose this extended error info returned by the service method?

Reply:
As far as I know we try to simplify these things as much as possible and see if there is a real usecase for consuming these objects programmatically.

If the only thing that developers would reasonably do is to print it to console we should expose error as a string.

The other way to think about it is the Operation behavior. Current guidelines require .Value call to throw if the operation failed. So maybe exposing this value only as exception message for that cause is enough.

cc @maririos who did a lot of designs around exposing errors and error details.

Reply:
Yes. We faced something similar in FR and ended up creating `RequestFailedException` and saving it in the Operation class so we could throw it when `.Value` was called. 
See https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/src/CopyModelOperation.cs#L58